q2_amr/amrfinderplus/database.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+
+from qiime2.util import duplicate
+
+from q2_amr.amrfinderplus.types import AMRFinderPlusDatabaseDirectoryFormat
+from q2_amr.card.utils import run_command
+
+
+def fetch_amrfinderplus_db() -> AMRFinderPlusDatabaseDirectoryFormat:
+    amrfinderplus_db = AMRFinderPlusDatabaseDirectoryFormat()
+
+    # Run AMRFinderPlus u function that downloads the database
+    run_amrfinder_u()
+
+    # Define path where the database will be downloaded to
+    conda_prefix = os.getenv("CONDA_PREFIX")
+    amrfinder_db_path = os.path.join(
+        conda_prefix, "share", "amrfinderplus", "data", "latest"
+    )
```

Comment:
It seems that amrfinder stores the database in the environment directory. Then when it is used the user doesn't need to specify the location. Is it ok to copy in that way from this path? It will always be the same?

@misialq what do you think about that?

Reply:
I just checked how they determine the path in the amrfinderplus package.
And they do it exactly the same, so I assume that this has to be a robust solution.

```
      if (const char* s = getenv("CONDA_PREFIX")) {
        defaultDb = string (s) + "/share/amrfinderplus/data/latest";
```

https://github.com/ncbi/amr/blob/a826b72520f22f858e4d0c9976ce07f01b5d557d/amrfinder.cpp#L641